MediaBrowser.Model/Configuration/ServerConfiguration.cs

```diff
@@ -264,5 +264,7 @@ public ServerConfiguration()
         /// </summary>
         /// <value>The limit for parallel image encoding.</value>
         public int ParallelImageEncodingLimit { get; set; }
+
+        public TrickplayOptions TrickplayOptions { get; set; } = new TrickplayOptions();
```

Comment:
Documentation on this would be nice to have (and be consistent with the rest of the file)